### ISSUE TITLE
Transparent background for text set to a black background in PowerShell

### ISSUE THREAD

**Issue by Johbii** [microsoft/terminal#9342]
# Environment

```none
Windows build number: 10.0.18363.778
Windows Terminal version (if applicable): 1.6.10571.0

Any other software?
Powershell version: 5.1.18362.752
```

# Steps to reproduce

1. Open a PowerShell instance in Windows Terminal
2. Enter `Write-Error -Message "anything"` (or otherwise receive an error/warning/any message with a black background)

# Expected behavior

![screenshot of PowerShell with background colors working](https://user-images.githubusercontent.com/30395979/109684877-9caff000-7b78-11eb-8354-b5087293f06c.PNG)

Critical error messages and warning messages are outlined with a black background to be more readable and noticeable compared with regular output (as is default with PowerShell).

# Actual behavior

![screenshot of Windows Terminal with colors not working](https://user-images.githubusercontent.com/30395979/109685001-ba7d5500-7b78-11eb-9c12-0e4b35901200.PNG)

Any text highlighted with a black background is made transparent/changed to the color of the terminal background.

Notice 40m is black on the old console host but not in Windows Terminal.

**Comment by j4james** [comment]
Quoting DHowett from issue https://github.com/microsoft/terminal/issues/9115#issuecomment-777715415:
> Unfortunately, this is by design. PowerShell erroneously tells us to use a black background _so often_ that it would cause there to be black streaks across the display (even if the terminal background was set to white or acrylic transparency.)
> 
> We had to disable black for PowerShell, specifically. #6810. Tracking the "fix" in /dup #6807.

**Comment by Johbii** [comment]
Oh OK, thanks for responding. Sorry for the dupe I wasn't aware of the issue.
